tests/testthat.R

```diff
@@ -1,13 +1,12 @@
 library(testthat)
 library(rsconnect)
 
-# record whether the configuration directory exists
-configDir <- rsconnect:::rsconnectConfigDir()
-configDirExists <- file.exists(configDir)
+# set temp config dir so the tests don't pollute it
```

Comment:
You might want to wrap some of this code in a function so that you can clean up in exit handlers via `on.exit()`. IIRC `test_check()` now actually throws an R error if any tests fail, which means clean up you might place after that call won't actually run if any tests fail.

Reply:
Good idea, done!